onnxscript/irbuilder.py

```diff
@@ -397,20 +397,7 @@ def to_function_proto(self) -> onnx.FunctionProto:
             onnx.helper.make_opsetid(domain, version) for domain, version in opsets.items()
         ]
 
-        # attribute_proto is introduced in version onnx==1.13.0.
-        # If this attribute is available, onnx-script uses it to
-        # default values for attributes. The function has then two
-        # lists, one list for attributes without default values,
-        # another one for attributes with default values.
-        # If this *attribute_proto* is not available,
-        # all attributes with a default value are moved to the first
-        # list, default values are removed.
-        # TODO: remove this when onnx with attribute_proto is released.
-        if hasattr(onnx.FunctionProto, "attribute_proto"):
-            atts = self.attrs
-        else:
-            atts = self.attrs + [a.attr_proto.name for a in self.attr_protos]
```

Comment:
Should we keep this to maintain compatibility with onnx 1.13, as it is still the latest version in production and people will need to test onnxscript with it? @gramalingam